src/libs/ascent/ascent.cpp

```diff
@@ -201,20 +254,115 @@ Ascent::open(const conduit::Node &options)
           m_options["mpi_comm"] = options["mpi_comm"];
         }
 
-        if(m_options.has_path("messages") &&
-           m_options["messages"].dtype().is_string() )
+        Node echo_opts;
+        echo_opts["echo_threshold"] = "info";
+        echo_opts["ranks"] = "root";
+
+        // messages echoed to std out
+        if(m_options.has_path("messages"))
+        {
+            if(m_options["messages"].dtype().is_string())
+            {
+                std::string msgs_opt = m_options["messages"].as_string();
+                if( msgs_opt == "verbose")
+                {
+                    m_verbose_msgs = true;
+                    echo_opts["echo_threshold"] = "all";
+                }
+                else if(msgs_opt == "quiet")
+                {
+                    m_verbose_msgs = false;
+                    echo_opts["echo_threshold"] = "error";
```

Comment:
wasn't "messages: quiet" going to be "echo_threshold: none" ?

Reply:
that was my intent, but looking at the previous logic I found that `error ` matches existing behavior (the error messages were never muted)

Reply:
gotcha. sounds good!